GiniVision/Classes/GiniNavigationViewController.swift

```diff
@@ -42,7 +42,7 @@ internal extension GiniNavigationViewController {
     }
     
     override var supportedInterfaceOrientations : UIInterfaceOrientationMask {
-        return UIInterfaceOrientationMask.portrait
+        return UIDevice.current.userInterfaceIdiom == .pad ? .all : .portrait
```

Comment:
We have an `Extensions.swift` file, you should add `UIDevice.current.userInterfaceIdiom == .pad` (or the solution for detecting iPads) into a helper method (`isiPad()`?) there.